code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -795,3 +795,50 @@ GLOBAL_LIST_INIT(default_medbay_channels, list(
 /obj/item/radio/phone/medbay/New()
 	..()
 	internal_channels = GLOB.default_medbay_channels.Copy()
+
+/obj/item/radio/proc/attempt_send_deadsay_message(mob/subject, message)
+	return
+
+GLOBAL_LIST_EMPTY(deadsay_radio_systems)
```

Comment:
Please move this list declaration somewhere else, rather than having it stashed at line 802 of a large file. Is there a defines file or something where this could go?

Reply:
slapped on the top like the other defines